serverlessworkflow/modules/ROOT/pages/core/defining-an-input-schema-for-workflows.adoc

```diff
@@ -17,11 +19,36 @@ You can define a `dataInputSchema` as follows:
 }
 ----
 
-In the previous definition, the `schema` property is a URI, which holds the path to the JSON schema used to validate the workflow data input. The URI can be a classpath URI, a file, or an HTTP URL. If a classpath URI is specified, then the JSON schema file must be placed in the resources section of the project.
+In the previous definition, the `schema` property is a URI, which holds the path to the JSON schema used to validate the workflow data input. The URI can be a classpath URI, a file, or an HTTP URL. If a classpath URI is specified, then the JSON schema file must be placed in the resources section of the project or any other directory included in classpath.
+
+`failOnValidationErrors` is an optional flag that indicates the behaviour adopted when the input data does not match the specified JSON schema. If not specified or set to true, an exception will be thrown and flow execution fails. If set to false, the flow will be executed and a log of level WARN with the validation errors will be printed.
+
+== Output Schema
+
+Serverless Workflow specification does not support Json Output schema until version 0.9. Therefore {product_name} is implementing it as an link:{spec_doc_url}#extensions[Serverless Workflow specification extension]. Output schema is applied after workflow execution to verify that the output model has the expected format. It is also useful for Swagger generation purposes. 
+
+As with Input schema, you should specify the url to the json schema, using `outputSchema` as follows
+
+.`outputSchema` definition
+----
+"extensions" : [ {
+      "extensionid": "workflow-output-schema",
+      "outputSchema": { 
+         "schema" : "URL_to_json_schema",
+          "failOnValidationErrors": false
+     }
+  ]
+----
+
+Same rules described for `dataInputSchema` applies for `schema` and `failOnValidationErrors`. The difference being that the latter flag is applied after workflow execution.
+
+== Example
```

Comment:
Is this an example with `dataInputSchema` and `outputSchema`? If yes, then you can mention it in the title. Something like this?
```suggestion
== Example with `dataInputSchema` and `outputSchema`
```